doc/en/user/source/data/app-schema/tutorial.rst

```diff
@@ -80,7 +80,7 @@ For ``gsml:MappedFeature`` the file is ``workspaces/gsml/gsml_MappedFeature/data
         </connectionParameters>
     </dataStore>
 
-.. note:: Ensure that there is no whitespace inside an ``entry`` element.
+.. note:: Ensure that there is no blank-space inside an ``entry`` element.
```

Comment:
I think we are stuck with whitespace characters for a bit, part of the unicode standard etc. I spent some time researching and could not find any evidence of white space (a visual design concept) being considered in a racist context.

Reply:
I just made the change due to its proximity to the "safe space" term.